test/svm/SvmSpoke.Bundle.ts

```diff
@@ -317,36 +332,29 @@ describe("svm_spoke.bundle", () => {
 
       // Verify valid leaf
       await program.methods
-        .executeRelayerRefundLeaf(stateAccountData.rootBundleId, leaf, proofAsNumbers)
-        .accounts({
-          state: state,
-          rootBundle: rootBundle,
-          signer: owner,
-          vault: vault,
-          tokenProgram: TOKEN_PROGRAM_ID,
-          mint: mint,
-          transferLiability,
-          systemProgram: anchor.web3.SystemProgram.programId,
-        })
+        .executeRelayerRefundLeaf(stateAccountData.rootBundleId, convertLeafIdToNumber(leaf), proofAsNumbers)
+        .accounts(executeRelayerRefundLeafAccounts)
         .remainingAccounts(wrongRemainingAccounts)
         .rpc();
-    } catch (err) {
+    } catch (err: any) {
       assert.include(err.toString(), "Account not found");
     }
 
     // Verify valid leaf
+    executeRelayerRefundLeafAccounts = {
```

Comment:
this block is repeated many times in this test file. we could re-use it at the top to decrease repetition in these tests & re-assign the values that change.

can save it for a follow on RP though.